app/routes/__index/certificate/information.tsx

```diff
@@ -22,90 +22,81 @@ export default function CertificateInstructionsRoute() {
   return (
     <Flex flexDirection="column" paddingBottom="20" maxWidth={750}>
       <Heading as="h1" size={{ base: 'lg', md: 'xl' }} mt={{ base: 6, md: 12 }} mb="4">
-        What is an SSL Certificate?
+        SSL Certificates
       </Heading>
       <Box>
-        <Text>
-          An SSL Certificate is like a digital lock that keeps information exchanged between your
-          computer and a website private and secure. It helps protect your sensitive information
-          like credit card numbers, login credentials, and other personal information from being
-          intercepted by hackers or cybercriminals.
-          <br />
-          <br />
-          When a website has an SSL certificate, you'll see a padlock icon and the web address
-          changes from "http" to "https" in your browser's address bar. This means that the
-          connection between your computer and the website is encrypted and secure.
-          <br />
-          <br />
-          SSL certificates are essential for websites that collect sensitive information from users,
-          like online stores, banks, or any site that requires you to enter personal information.
-          They help build trust between websites and their users and ensure that your data is kept
-          private and secure.
-        </Text>
-
-        <Heading as="h3" size="md" marginTop="5">
-          Browser Security Warnings
+        <Heading as="h2" size="md" marginTop="5">
+          What is an SSL Certificate?
         </Heading>
-
-        <Text marginTop="3">
+        <Text marginY="1em">
```

Comment:
I am curious why you decided to use "1em" instead of "4" which I think is the same thing in terms of it's value. Probably would be better to use units we use everywhere else

Reply:
Update: "4" is not 1em, it is 1rem, which as far as I understand doesn't make any difference in the context of our app

Reply:
I can switch to use `{4}` if you want, sure.